changelog/20.0/20.0.0/release_notes.md

```diff
@@ -360,8 +360,12 @@ You can now run `VDiff`s on OnlineDDL schema change migrations, which are not ye
 
 VTTablet exposes two new counter stats:
 
- * `QueryCacheHits`: Query engine query cache hits
- * `QueryCacheMisses`: Query engine query cache misses
+ * `QueryCacheHits`: Query engine query plan cache hits
+ * `QueryCacheMisses`: Query engine query plan cache misses
+
+> [!NOTE]
+> `QueryCache` does not refer to a query cache, but to a query plan cache.
+> In v21, these metrics will be deprecated and replaced by `TabletQueryPlanCacheHits` and `TabletQueryPlanCacheMisses`.
```

Comment:
The reason metric names are not prefixed with `Tablet` or `Gate` is that the binary name is automatically added to the metric name as a prefix. So this becomes `vttablet_query_plan_cache_misses` if you look at `/metrics`.
All of the metrics introduced in this PR need to be renamed and the `Tablet` prefix should be removed.

Reply:
That won't work as the vtgate and vttablet metrics will have the same name in vtcombo, leading to a panic/failure.

Reply:
I will have to see how I can tweak vtcombo